examples/AsyncTranscribeLocalMediaFile.java

```diff
@@ -35,7 +35,8 @@ public static void main(String[] args) {
     revAiJobOptions.setRemoveDisfluencies(true);
     revAiJobOptions.setSpeakerChannelsCount(null);
     revAiJobOptions.setDeleteAfterSeconds(2592000); // 30 days in seconds
-    revAiJobOptions.setLanguage("en");
+    revAiJobOptions.setLanguage("en"); // Specify language using the one of the supported ISO 639-1 (2-letter) or
+                                       // ISO 639-3 (3-letter) language codes as defined in the API Reference
```

Comment:
I don’t think these are the places we need to add the inline comments - looking up `language` in the repo, there’s a `RevAiJobOptions` that contain the parameter for `language` and I that is where we should be updating the documentation.